### PR TITLE
write up social media campaigns workflow

### LINKED ISSUE #109: document our social media accounts and how best to use them
I'll get a page set up for this with a list and some ideas and @sandyklc you can follow up and flesh it out? :)

### PR DESCRIPTION
**Pull request by bobheadxi**
### Related Tickets

Closes #109

### Changes

Documents some ideas from [today's meeting](https://docs.google.com/document/d/13C-t5RGA94SmmgcNgpk_DsWwwiODg1kxmSyq3guIAOs/edit#heading=h.cuxcnoqky35h) - would like to hear what everyone thinks!

Rendered: https://deploy-preview-137--ubclaunchpad-docs.netlify.app/handbook/tools/social-media

### Checklist

* I've read the [Structuring Content](https://docs.ubclaunchpad.com/CONTRIBUTING#structuring-content) guide
* I've read up on our [Pull Request checks](https://docs.ubclaunchpad.com/CONTRIBUTING#checks)

### CODE REVIEW

**Comment by sandyklc** [comment]
[State of Launch Pad 2020 Social Media Posts](https://docs.google.com/document/d/1bFrQMr1UVYN6LI-jRZpAw_Gt2VZ5Ty9L0Bp_VksYdg0/edit#)

The first post will be July 19, 2020 (Sunday) at 5pm. 

Let us know of any comments before we post!

File: handbook/tools/social-media.md [hunk 1]
```diff
@@ -8,26 +8,60 @@ Any Launch Pad original content could make good social media posts to help us sh
...
+
+#### Mini-posts
+
+Once the article is prepared, segment it out into a series of mini-posts that can be adapted to work as Facebook and Instagram posts.
 
-TODO
+Each of these should generally target a specific section of the article, with:
 
-### Instagram
+* A brief blurb for the relvant section of the article
```
Comment (tarikeshaq) [inline handbook/tools/social-media.md:43]:
```suggestion
* A brief blurb for the relevant section of the article
```

File: handbook/tools/social-media.md [hunk 2]
```diff
@@ -8,26 +8,60 @@ Any Launch Pad original content could make good social media posts to help us sh
...
+## Social Media Campaigns
+
+Outside of occasionally sharing miscellaneous tidbits from [`#ot-sharing`](https://ubclaunchpad.slack.com/archives/C01622TSU9W), we should focus on creating social media posts as part of campaigns, each centered around a long-form article. This helps us have consistent messaging and (hopefully) higher-quality, more engaging content.
+
+### Preparation
+
+Decide on a start date and topic for the social media campaign. A preparation workflow could look like:
+
+1. [Create a Medium article](#medium-articles) (finish 5 days before start date)
+2. [Split article into mini-posts](#mini-posts) (start after Medium article, finish 1 day before start date)
```
Comment (tarikeshaq) [inline handbook/tools/social-media.md:28]:
```suggestion
2. Split the article into [mini-posts](#mini-posts) (start after creating the Medium article, finish 1 day before start date)
```

File: handbook/tools/social-media.md [hunk 3]
```diff
@@ -8,26 +8,60 @@ Any Launch Pad original content could make good social media posts to help us sh
...
+3. [Write a newsletter issue](#newsletter) (start after Medium article, finish 3 days before start date)
```
Comment (tarikeshaq) [inline handbook/tools/social-media.md:29]:
```suggestion
3. [Write a newsletter issue](#newsletter-issues) (start after creating the Medium article, finish 3 days before start date)
```

File: handbook/tools/social-media.md [hunk 4]
```diff
@@ -8,26 +8,60 @@ Any Launch Pad original content could make good social media posts to help us sh
...
-TODO
+### Publishing
 
-### Newsletter
+On the start date, a publishing schedule could look like:
 
-We currently use [Buttondown](https://buttondown.email/) as our newsletter provider. The newsletter should primarily be used to provide updates (when applications open, club events, etc) or to provide highlights of past or recent Launch Pad content (such as Medium articles or Instagram posts).
+1. Publish the [Medium article](#medium-articles)
+2. Share the Medium article with a brief blurb on LinkedIn
+3. Send out the [newsletter issue](#newsletter)
```
Comment (tarikeshaq) [inline handbook/tools/social-media.md:66]:
```suggestion
3. Send out the [newsletter issue](#newsletter-issues)
```

File: handbook/tools/social-media.md [hunk 5]
```diff
@@ -8,26 +8,78 @@ Any Launch Pad original content could make good social media posts to help us sh
...
-Credentials are available in the [Exec repository](https://github.com/ubclaunchpad/exec) - please reach out to [`#ask-leads`](https://ubclaunchpad.slack.com/archives/CK935RD3Q) to get in touch with one of the presidents if you are interested in helping us run our social media accounts!
+* [https://ubclaunchpad.com/facebook](https://ubclaunchpad.com/facebook)
+* [https://ubclaunchpad.com/instagram](https://ubclaunchpad.com/instagram)
+* [https://ubclaunchpad.com/medium](https://ubclaunchpad.com/medium)
 
-### Facebook
+Credentials for accounts are available in the [Exec repository](https://github.com/ubclaunchpad/exec) - please reach out to [`#ask-leads`](https://ubclaunchpad.slack.com/archives/CK935RD3Q) to get in touch with one of the presidents if you are interested in helping us run our social media accounts!
 
-TODO
+Please reach out to the presidents to create accounts for social media or third-party services so that we can keep track of them. Also see our [email documentation](./email.md).
```
Comment (andrewzulaybar) [inline handbook/tools/social-media.md:19]: Just want to make sure that `email.md` isn't implemented as of yet?

File: handbook/tools/social-media.md [hunk 6]
```diff
@@ -8,26 +8,78 @@ Any Launch Pad original content could make good social media posts to help us sh
...
+All content prepared for social media campaigns should be stored or linked to in the [Social Media](https://drive.google.com/drive/folders/1tRSa-obWwr_JU57tpscqmxIUZPlfqckQ?usp=sharing) folder (under Strategy).
 
-TODO
+### Preparation
 
-### Newsletter
+Decide on a start date and topic for the social media campaign. A preparation workflow could look like:
 
-We currently use [Buttondown](https://buttondown.email/) as our newsletter provider. The newsletter should primarily be used to provide updates (when applications open, club events, etc) or to provide highlights of past or recent Launch Pad content (such as Medium articles or Instagram posts).
+1. [Create a Medium article](#medium-articles) (finish 5 days before start date)
```
Comment (andrewzulaybar) [inline handbook/tools/social-media.md:35]:
```suggestion
1. Create a [Medium article](#medium-articles) (finish 5 days before start date)
```

File: handbook/tools/social-media.md [hunk 7]
```diff
@@ -8,26 +8,78 @@ Any Launch Pad original content could make good social media posts to help us sh
...
+2. Split the article into [mini-posts](#mini-posts) (start after creating the Medium article, finish 1 day before start date)
+3. [Write a newsletter issue](#newsletter-issues) (start after creating the Medium article, finish 3 days before start date)
```
Comment (andrewzulaybar) [inline handbook/tools/social-media.md:37]:
```suggestion
3. Write a [newsletter issue](#newsletter-issues) (start after creating the Medium article, finish 3 days before start date)
```

File: handbook/tools/social-media.md [hunk 8]
```diff
@@ -8,26 +8,78 @@ Any Launch Pad original content could make good social media posts to help us sh
...
+* Brief summary of what's coming up next
+
+A good tool for drafting newsletter issues in Markdown is [hackmd.io](https://hackmd.io/) - you can log in using the UBC Launch Pad Google account.
+
+#### Short Links
+
+In lieu of a link shortener, we use [redirects (short links)](https://ubclaunchpad.com/config/#adding-a-custom-redirect) for paths on `ubclaunchpad.com` (for example, `ubclaunchpad.com/medium`). We maintain a redirect, `ubclaunchpad.com/latest`, that should always point to the latest campaign's [Medium article](#medium-articles).
+
+::: warning
+The `ubclaunchpad.com/latest` short link **should only be used in Instagram posts and for sharing directly with people** - we don't want links in longer-standing content (for example, Facebook/LinkedIn/newsletter posts) to become outdated over time!
```
Comment (andrewzulaybar) [inline handbook/tools/social-media.md:73]:
```suggestion
The `ubclaunchpad.com/latest` short link **should only be used in the Instagram profile's website field and for sharing directly with people** - we don't want links in longer-standing content (for example, Facebook/LinkedIn/newsletter posts) to become outdated over time!
```

Or something along those lines, just to clarify that they shouldn't be used in IG posts (cause then they will eventually be outdated)

File: handbook/tools/social-media.md [hunk 9]
```diff
@@ -8,26 +8,78 @@ Any Launch Pad original content could make good social media posts to help us sh
...
+### Publishing
+
+On the start date, a publishing schedule could look like:
+
+1. Publish the [Medium article](#medium-articles)
+2. Share the Medium article with a brief blurb on LinkedIn (this should be the only LinkedIn post)
+3. Send out the [newsletter issue](#newsletter-issues)
+4. Update the [`/latest` short link](#short-links)
+5. Post an update in [`#ot-random`](https://ubclaunchpad.slack.com/archives/C061Q5328) about the new campaign
+6. On 2-day intervals, publish the [mini-posts](#mini-posts) as Facebook and Instagram posts - for example, at +0 days, +2 days, +4 days, etc.
```
Comment (andrewzulaybar) [inline handbook/tools/social-media.md:85]:
```suggestion
6. On 2-day intervals, publish the [mini-posts](#mini-posts) as Facebook and Instagram posts - for example, at +0 days, +2 days, +4 days, etc.
7. **[Optional]:** Post a Facebook or Instagram story promoting the recently published post
```

File: handbook/tools/social-media.md (hunk 5 above)
Comment (bobheadxi) [reply handbook/tools/social-media.md:19]: yep this should be fine, the email docs are still pending review over at https://github.com/ubclaunchpad/docs/pull/138

File: handbook/tools/social-media.md (hunk 9 above)
Comment (bobheadxi) [reply handbook/tools/social-media.md]: d236d0a3db8073f32b01699380ec6472ae657161

File: handbook/tools/analytics.md [hunk 10]
```diff
@@ -23,3 +23,7 @@ Using the site code, you can now set up analytics on your project! Note that we
 * If you need more granular control, you can use [`fathom-client`](https://github.com/derrickreimer/fathom-client) instead of the embedding the tracker script - see [example usages in Launch Pad](https://sourcegraph.com/search?q=repo:%5Egithub.com/ubclaunchpad/*+Fathom.load%28&patternType=literal).
   * for [Vue.js](https://vuejs.org/) projects, you can use [`@ubclanchpad/vue-fathom`](https://github.com/ubclaunchpad/vue-fathom)
   * for [Vuepress](https://vuepress.vuejs.org/) projects, you can use [`@ubclaunchpad/vuepress-plugin-fathom`](https://github.com/ubclaunchpad/vuepress-plugin-fathom)
+
+## Social Media
+
+[Facebook](https://www.facebook.com/ubclaunchpad/insights/), [Medium](https://medium.com/ubc-launch-pad-software-engineering-blog/stats/overview), and [Buttondown (newsletter)](https://buttondown.email/analytics) have built-in analytics. We also have analytics set up for a few of our sites, linked above.
```
[the remark this answers is not in the document]
Comment (bobheadxi) [reply handbook/tools/analytics.md]: ea002b1f1f671aefa821e8324d5abf5418d70890

File: handbook/tools/social-media.md (hunk 8 above)
Comment (bobheadxi) [reply handbook/tools/social-media.md:73]: this sounds like a safer policy!

**Comment by bobheadxi** [comment]
Thanks @andrewzulaybar ! 🚀

**Comment by bobheadxi** [comment]
@sandyklc I'm going to merge this for now - if you need to make any adjustments please send follow-up PRs :) Thanks everyone for reviewing!!

**Pull request merged**